lib/src/summarize.dart

```diff
@@ -4,23 +4,235 @@
 
 library dart_pad.summarize;
 
+import 'package:crypto/crypto.dart';
+
 import '../services/dartservices.dart';
 
 /// Instances of this class take string input of dart code as well as an analysis result,
 /// and output a text description ofthe code's size, packages, and other useful information.
 class Summarizer {
   _SummarizeToken storage;
-
+  String _code;
+  String _css;
+  String _html;
   bool resultsPresent;
-
-  Summarizer(String input, [AnalysisResults analysis]) {
-    if (input == null) throw new ArgumentError("Input can't be null.");
+  int randomizer;
+  
+  static Map<String, List<int>> cuttoffs = {
+    'size': [8, 30, 1000], //0-7 = small, 8 - 29 = decent, 29+ = gigantic
+    'errorCount': [1,5,100]
+  };
+  
+  static Map<String, String> codeKeyWords = {
+    'await' : 'await',
+    'async' : 'async',
+    'rpc' : 'RESTful serverside app'
+  };
+  
+  static Map<String, String> additionKeyWords = {
+      'pirate' : 'pirates',
+      'bird' : 'birds',
+      'llama' : 'llamas',
+  };
+    
+  static Map<String, List<String>> categories = {
+    /// This [size] [codeQuantifier] contains [error] errors and warnings.
+    'size-2': ['gigantic', 'Jupiterian sized', 'immense', 
+      'massive', 'enormous', 'huge', 'epic', 'humongous'],
+    'size-1': ['decently sized', 'exceptional', 'awesome', 
+      'amazing', 'visionary', 'legendary'],
+    'size-0': ['itty-bitty', 'miniature', 'tiny', 'pint-sized'],
+    'compiledQuantifier': ['DART program', 'pad'],
+    'failedQuantifier': ['assemblage of characters', 'series of strings', 'grouping of letters'],
+    'errorCount-2':['many', 'a motherload of', 'copious amounts of', 'unholy quantities of'],
+    'errorCount-1':['some', 'a few', 'sparse amounts of', 'very few instances of'],
+    'errorCount-0':['zero', 'no', 'a nonexistent amount of', '0.0000', '0'],
+    'use' : ['demonstrates', 'contains', 'illustrates', 'depicts'],
+    'code-0': ['it'],
+    'code-1': ['It'],
+  };
+  
+  Summarizer(String input, {String html, String css, AnalysisResults analysis}) {
```

Comment:
Let's change `input` to a named param to match the other parameters. The constructor would look something like this: `Summarizer({this.dart, this.html, this.css, AnalysisResults.analysis) {...`.

The `this.` notation tells the runtime to assign the given param to the matching instance field. It lets you have a final field assigned from a ctor w/ using very terse syntax.